lighthouse-core/test/audits/dobetterweb/external-anchors-use-rel-noopener-test.js

```diff
@@ -11,6 +11,8 @@ const assert = require('assert').strict;
 
 const URL = 'https://google.com/test';
 
+const node = {};
```

Comment:
feels a little too global.

you could move to L121

and i guess just use an empty `{}` literal on L112.